geopyspark/geopycontext.py

```diff
@@ -85,34 +85,17 @@ def map_key_input(key_type, is_boundable):
             else:
                 raise Exception("Could not find key type that matches", key_type)
 
-    def create_schema(self, key_type):
-        """Creates an AvroSchema.
+    def create_tuple_serializer(self, key_type):
+        decoder = self.protobufregistry.create_partial_tuple_decoder(key_type=key_type)
+        encoder = self.protobufregistry.create_partial_tuple_encoder(key_type=key_type)
 
-        Args:
-            key_type (str): The type of the ``K`` in the tuple, ``(K, V)`` in the RDD.
-
-        Returns:
-            An AvroSchema for the types within the RDD.
-        """
-
-        return self._jvm.geopyspark.geotrellis.SchemaProducer.getSchema(key_type)
-
-    def create_tuple_serializer(self, schema, key_type, value_type):
-        decoder = \
-                self.avroregistry.create_partial_tuple_decoder(key_type=key_type,
-                                                               value_type=value_type)
-
-        encoder = \
-                self.avroregistry.create_partial_tuple_encoder(key_type=key_type,
-                                                               value_type=value_type)
-
-        return AutoBatchedSerializer(AvroSerializer(schema, decoder, encoder))
+        return AutoBatchedSerializer(ProtoBufSerializer(decoder, encoder))
 
-    def create_value_serializer(self, schema, value_type):
-        decoder = self.avroregistry._get_decoder(value_type)
-        encoder = self.avroregistry._get_encoder(value_type)
+    def create_value_serializer(self, value_type):
```

Comment:
Now that it is no longer possible or needed to register your own serializer these three methods into `protobufserializer.py`